Dockerfile

```diff
@@ -44,7 +38,7 @@ ENV PATH="/app/flutter/bin:$PATH"
 
 # Used if wanting to build the container with a different branch
 # e.g. `make FLUTTER_BRANCH=dev build`
-# This is not to be confused with the $TEST_TARGET_CHANNEL
+# This is not to be confused with the $FLUTTER_TEST_BRANCH
 RUN if test -n "$FLUTTER_BRANCH" -a "$FLUTTER_BRANCH" != "stable" ; then \
```

Comment:
This has been around for a while but I'll just call it out - it might be better to make it `make FLUTTER_BRANCH=beta build` since dev doesn't exist anymore.